admin/class-form.php

```diff
@@ -62,7 +62,7 @@ public function admin_header() {
 			 */
 			require_once( ABSPATH . 'wp-admin/options-head.php' );
 			?>
-        	<h1><?php _e( 'Wired Impact Volunteer Management: Help & Settings', 'wired-impact-volunteer-management' ); ?></h1>
+        	<h1><?php _e( 'Wired Impact Volunteer Management: ' . apply_filters( 'wivm_help_and_settings_menu_item_name', 'Help & Settings' ), 'wired-impact-volunteer-management' ); ?></h1>
```

Comment:
What would you think about making this filter more generic to be something like `wivm_submenu_page_name` (to be in line with `wivm_submenu_pages`)  and adding it so it can be used on the "Volunteers" and the "Volunteer" admin subpages as well? It seems a little funny to me that right now we're creating a filter for one specific use.

Reply:
I think that's a great idea. I'll move forward with getting this updated.